src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolJudicialOfficeMappingSSCS.java

```diff
@@ -0,0 +1,633 @@
+
+import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.kie.api.runtime.rule.QueryResults;
+import org.kie.api.runtime.rule.QueryResultsRow;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.JudicialAccessProfile;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.RoleAssignment;
+import uk.gov.hmcts.reform.orgrolemapping.helper.TestDataBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static uk.gov.hmcts.reform.orgrolemapping.domain.service.RequestMappingService.ROLE_ASSIGNMENTS_RESULTS_KEY;
+
+
+@RunWith(MockitoJUnitRunner.class)
+class DroolJudicialOfficeMappingSSCS extends DroolBase {
+
+    @Test
+    void shouldReturnTribunalJudgeSalariedRoles() {
+
+        judicialAccessProfiles.forEach(judicialAccessProfile -> {
+            judicialAccessProfile.setAppointment("President of Tribunal");
+            judicialAccessProfile.setAppointmentType("Salaried");
+            judicialAccessProfile.setServiceCode("BBA3");
+            judicialAccessProfile.getAuthorisations().forEach(a -> a.setServiceCode("BBA3"));
+        });
+
+        //Execute Kie session
+        buildExecuteKieSession(getFeatureFlags("sscs_wa_1_0", true));
+
+        //Extract all created role assignments using the query defined in the rules.
+        List<RoleAssignment> roleAssignments = new ArrayList<>();
+        QueryResults queryResults = (QueryResults) results.getValue(ROLE_ASSIGNMENTS_RESULTS_KEY);
+        for (QueryResultsRow row : queryResults) {
+            roleAssignments.add((RoleAssignment) row.get("$roleAssignment"));
+        }
+
+        //assertion
+        assertFalse(roleAssignments.isEmpty());
+        assertEquals(4, roleAssignments.size());
+        roleAssignments.forEach(r -> {
+            assertEquals(judicialAccessProfiles.stream().iterator().next().getUserId(), r.getActorId());
+        });
+        assertEquals("caseworker-sscs-judge", roleAssignments.get(0).getRoleName());
+        assertEquals("case-allocator", roleAssignments.get(1).getRoleName());
+        assertEquals("task-supervisor", roleAssignments.get(2).getRoleName());
+        assertEquals("hmcts-judiciary", roleAssignments.get(3).getRoleName());
+        assertEquals("Salaried", roleAssignments.get(0).getAttributes().get("contractType").asText());
+        assertEquals("Salaried", roleAssignments.get(1).getAttributes().get("contractType").asText());
+        assertEquals("Salaried", roleAssignments.get(2).getAttributes().get("contractType").asText());
+        assertEquals("Salaried", roleAssignments.get(3).getAttributes().get("contractType").asText());
+    }
```

Comment:
Please asset jurisdiction, worktypes, authorisations, primarylocations as well

Reply:
Added the missing assertions.